test/extensions/firstorder/firstorder_settings.py

```diff
@@ -0,0 +1,153 @@
+"""Test configurations for `backpack.core.extensions.firstorder` methods
+
+
```

Comment:
Can you add a comment that these settings will be shared among all first-order executions?